build.gradle

```diff
@@ -193,8 +208,13 @@ sonarqube {
         property "sonar.jacoco.itReportPath", "${project.buildDir}/jacoco/integration.exec"
         property "sonar.pitest.mode", "reuseReport"
         property "sonar.pitest.reportsDirectory", "build/reports/pitest"
-        property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/juddata/**," +  //@To do need to cover sonar coverage for camel routes
-                "src/main/java/uk/gov/hmcts/reform/juddata/JudicialApplication.java"
+
+        property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/juddata/configuration/**," + //To do remove unused code
+                "src/main/java/uk/gov/hmcts/reform/juddata/JudicialApplication.java," +
+                "src/main/java/uk/gov/hmcts/reform/juddata/controller/**," +
+                "src/main/java/uk/gov/hmcts/reform/juddata/orchestrator/**," +
+                "src/main/java/uk/gov/hmcts/reform/juddata/scheduler/**," +
+                "src/main/java/uk/gov/hmcts/reform/juddata/service/**"
     }
 }
 
```

Comment:
why is this not corrected?

Reply:
seems adding + new line cause sonar fails

Reply:
do we need to exclude this? "src/main/java/uk/gov/hmcts/reform/juddata/scheduler/**